deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -139,11 +139,8 @@ def compute(self, context: Context) -> CheckResult:
         df_train = pd.DataFrame(self._train_properties)
         df_test = pd.DataFrame(self._test_properties)
         if len(df_train) < self.min_samples or len(df_test) < self.min_samples:
-            return CheckResult(
-                value=None,
-                display=f'Not enough samples to calculate drift score, min {self.min_samples} samples required.',
-                header='Image Property Drift'
-            )
+            raise NotEnoughSamplesError(f'Not enough samples to calculate drift score, min {self.min_samples} samples required'
```

Comment:
1. Don't use shotrcuts ("minimum" instead of "min")
2. Better to also hint on solution, add in the end "Use 'min_samples' parameter to change the requirement"